dandi/bids_utils.py

```diff
@@ -0,0 +1,42 @@
+from typing import Optional
+
+import click
+
+from .utils import pluralize
+
+
+def evaluate_validation(
+    validation_result: dict,
+    allow_errors: Optional[bool] = False,
```

Comment:
`Optional[...]` means "can be `None`", not "is an optional argument."  It does not seem to apply here.